solr/core/src/test/org/apache/solr/cloud/TestRebalanceLeaders.java

```diff
@@ -445,10 +449,12 @@ private void rebalancePropUsingStandardRequest(String prop)
     if (prop.toLowerCase(Locale.ROOT).contains("preferredleader") == false) {
       params.set("shardUnique", true);
     }
-    QueryRequest request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-    QueryResponse resp = request.process(cluster.getSolrClient());
-    assertEquals("Call to rebalanceLeaders failed ", 0, resp.getStatus());
+    var request =
+        new GenericSolrRequest(METHOD.GET, "/admin/collections", SolrRequestType.ADMIN, params);
+    request.setRequiresCollection(false);
+    SimpleSolrResponse resp = request.process(cluster.getSolrClient());
```

Comment:
minor but declaring as SimpleSolrResponse is overly specific.  I'd just use `var`

Reply:
Done